### PR TITLE
Task login and register

### PR DESCRIPTION
**Pull request by hhadzic3**

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -1,28 +1,120 @@
...
+                    className='remember'
+                    control={< Checkbox value = "remember" color = "primary" />}
+                    label="Remember me"/>
+                <Button type="submit" fullWidth variant="contained" className='submit'>
+                    Log In
+                </Button>
+                <Grid className='end' container>
+                    <Grid item xs>
+                        <Link className='link' to="/home" variant="body2">
+                            Forgot password?
```
Comment (eminahuskic) [inline src/components/Login/Login.js]: If it is not implemented, no need to have it here.

File: src/components/Profile/TabPanel.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    const handleChange = (event, newValue) => {
+        setValue(newValue);
+    };
+    
+    function createData(Name, Price, Label) {
+        return { Name, Price, Label };
+      }
+      
+      const rows = [
+        createData('Frozen yoghurt', 159, "Top rated"),
```
Comment (eminahuskic) [inline src/components/Profile/TabPanel.js:61]: Why hardcoding?

File: src/components/Signup/Signup.js [hunk 3]
```diff
@@ -9,15 +11,186 @@ import Container from '@material-ui/core/Container';
...
+          last_name: this.state.last_name,
+          email: this.state.email,
+          password: this.state.password,
+          billing_address: this.state.billing_address,
+          shipping_address: this.state.shipping_address,
+          phone: this.state.phone,
+          country: this.state.country
+        }
+    
+        register(newUser).then(res => {
```
Comment (eminahuskic) [inline src/components/Signup/Signup.js]: what if register fails?

File: src/components/Profile/TabPanel.js (hunk 2 above)
Comment (hhadzic3) [reply src/components/Profile/TabPanel.js:61]: This is temporary until functionality is implemented.

**Review by eminahuskic — APPROVED** [review]

**Pull request merged**